src/pranitaPJun20/IntersectionOfTwoArrays.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Program 2: Find intersection(common elements) of two arrays.
+Input: arr1 -> {10.45, 14.0, 18.35, 88.88, 54.10, 18.35}
+          arr2 -> {17.20, 13.30, 10.45, 18.35, 84.33, 13.30}
+
+Output: 10.45, 18.35
+Method Signature : double[] getIntersectionOfArray(double[] arr1, double[] arr2)
+ */
+package pranitaPJun20;
+
+public class IntersectionOfTwoArrays {
+	int countOfTempArray = 0;
+
+	public static void main(String[] args) {
+		double[] array1 = { 10.45, 14.0, 18.35, 88.88, 54.10, 18.35 };
+		double[] array2 = { 17.20, 13.30, 10.45, 18.35, 84.33, 13.30 };
+		new IntersectionOfTwoArrays().getIntersectionOfArray(array1, array2);
+	}
+
+	// Find out commomn elements from both the given arrays
+	void getIntersectionOfArray(double[] array1, double[] array2) {
+		double[] tempIntersectionArray = new double[(array1.length)];
```

Comment:
Instead of taking array1.length, you should first compare length of both arrays, whichever is small you should set that number as tempIntersectionArray length.

for example,
int length = array1.length;
if(array1.length<array2.length)
     length = array2.length

Reply:
Yes ,got it.Will implement it.

Reply:
Changes done.